blocks/footer-block/_index.scss

```diff
@@ -0,0 +1,51 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-footer {
+	&__links {
+		&-group {
+			&-list {
+				&-item {
+					@include scss.block-components("footer-links-group-list-item");
+					@include scss.block-properties("footer-links-group-list-item");
+				}
+
+				@include scss.block-components("footer-links-group-list");
+				@include scss.block-properties("footer-links-group-list");
+			}
+
+			@include scss.block-components("footer-links-group");
+			@include scss.block-properties("footer-links-group");
+		}
+
+		@include scss.block-components("footer-links");
+		@include scss.block-properties("footer-links");
+	}
+
+	&__social-links {
+		&-container {
+			@include scss.block-components("footer-social-links-container");
+			@include scss.block-properties("footer-social-links-container");
+		}
+
+		> *:first-child {
+			@include scss.block-components("footer-social-links-first");
+			@include scss.block-properties("footer-social-links-first");
```

Comment:
I see we are setting styles for last child but not first child. Is this for future proofing?